<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="BikramKesharis-Mac-mini.local" failures="1" tests="7" name="Test" time="72.885" errors="0" timestamp="2024-07-03T10:27:27 IST">
  <testcase classname="com.akhilsystem_release.test.emr.DoctorAssign" name="openAppointment" time="8.877"/>
  <testcase classname="com.akhilsystem_release.test.emr.DoctorAssign" name="fstDoctorAppointment" time="15.754"/>
  <testcase classname="com.akhilsystem_release.test.emr.DoctorAssign" name="closeAppointment" time="3.078"/>
  <testcase classname="com.akhilsystem_release.test.emr.DoctorAssign" name="fstDoctorLogin" time="20.267"/>
  <testcase classname="com.akhilsystem_release.test.emr.DoctorAssign" name="assignToDoctor" time="3.127"/>
  <testcase classname="com.akhilsystem_release.test.emr.DoctorAssign" name="assignedDocLogin" time="10.659"/>
  <testcase classname="com.akhilsystem_release.test.emr.DoctorAssign" name="verifySndDoctorData" time="2.967">
    <failure type="java.lang.AssertionError" message="expected [376569] but found [971316]">
      <![CDATA[java.lang.AssertionError: expected [376569] but found [971316]
at com.akhilsystem_release.test.emr.DoctorAssign.verifySndDoctorData(DoctorAssign.java:142)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at com.akhilsystem_release.test.emr.TestRunner.main(TestRunner.java:49)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- verifySndDoctorData -->
</testsuite> <!-- Test -->
